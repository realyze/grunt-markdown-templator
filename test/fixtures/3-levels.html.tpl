<% _.forEach(children, function(main) { %>

<div id="<%= main.id %>">

  <%= main.body %>

  <icon_nav/>

  <% _.forEach(main.children, function(section) { %>

    <section id="<%= section.id %>">
      <%= section.body %>

      <% _.forEach(section.children, function(article) { %>
        <article id="<%= article.id %>">
          <%= article.body %>
        </article>
      <% }) %>

    </section><% })
  %>

</div><%
}) %>
