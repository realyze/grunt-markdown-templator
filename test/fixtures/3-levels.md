# Solutions {foobar}

This is the Solutions noodle.

## Overview

This is the overview of the Solutions noodle.

## Web Apps

### Level 3 header
And some vague content...

This is my rant about Web Apps level 3

 * with bullet points
 * oh yes!
 * with many bullet points!!!


### Another level 3 header
And some more vague content...

## Mobile apps
Another rant about **mobile apps** goes _here_
