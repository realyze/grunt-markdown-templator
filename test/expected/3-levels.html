<div id="foobar">

    <h1>Solutions</h1>

    <p>This is the Solutions noodle.</p>

    <icon_nav/>



    <section id="overview">
        <h2>Overview</h2>

        <p>This is the overview of the Solutions noodle.</p>



    </section>

    <section id="web-apps">
        <h2>Web Apps</h2>


        <article id="level-3-header">
            <h3>Level 3 header</h3>

            <p>And some vague content...</p>

            <p>This is my rant about Web Apps level 3</p>

            <ul>
                <li>with bullet points</li>
                <li>oh yes!</li>
                <li>with many bullet points!!!</li>
            </ul>
        </article>

        <article id="another-level-3-header">
            <h3>Another level 3 header</h3>

            <p>And some more vague content...</p>
        </article>


    </section>

    <section id="mobile-apps">
        <h2>Mobile apps</h2>

        <p>Another rant about <strong>mobile apps</strong> goes <em>here</em>
        </p>



    </section>

</div>
